apps/emqx/src/emqx_session_router.erl

```diff
@@ -60,21 +56,30 @@
 
 -type(dest() :: node() | {group(), node()}).
 
--define(ROUTE_TAB, emqx_session_route).
+-define(ROUTE_RAM_TAB, emqx_session_route_ram).
+-define(ROUTE_DISC_TAB, emqx_session_route_disc).
 
 -define(SESSION_INIT_TAB, session_init_tab).
 
 %%--------------------------------------------------------------------
 %% Mnesia bootstrap
 %%--------------------------------------------------------------------
 
-mnesia(boot) ->
-    ok = mria:create_table(?ROUTE_TAB, [
+create_router_tab() ->
+    ok = mria:create_table(?ROUTE_DISC_TAB, [
```

Comment:
Q: shouldn't only one of the disc or ram tables be created conditioned on the `emqx_persistent_session:storage_type/0`, analogous to the [other tables](https://github.com/emqx/emqx/blob/46788ad31b737978f114652094ab664b90a98f1d/apps/emqx/src/emqx_persistent_session.erl#L88-L95)?
(Same [here](https://github.com/emqx/emqx/blob/46788ad31b737978f114652094ab664b90a98f1d/apps/emqx/src/emqx_trie.erl#L90))

Reply:
:thinking: Yes, that seems reasonable. The idea is that both set of tables should be able to coexist in the same schema, though, so that you can reconfigure a cluster without doing any schema changes.

I will give this some thought.